pkgs/development/compilers/kotlin-native/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  pname = "kotlin-native";
+  version = "1.5.21";
+
+  src = fetchurl {
+    url = "https://github.com/JetBrains/kotlin/releases/download/v${version}/kotlin-native-linux-${version}.tar.gz";
+    hash = "sha256-+j3+ycEXEcK3E6FIK8xFEbuPc/GC8Sqn2FiUP28IQ5c=";
+  };
+
+  buildInputs = [ makeWrapper ] ;
+
+  installPhase = ''
+    mkdir -p $out
+    mv * $out
+
+    for p in $(ls $out/bin); do
```

Comment:
I would be happier if we would list the binaries here to not wrap something by accident.